StudioCore/TaskLogger.cs

```diff
@@ -0,0 +1,179 @@
+﻿using ImGuiNET;
```

Comment:
I'm not against adding logging but I feel like we should use a proper logging framework that can handle things like different sinks/multithreading/buffered streaming rather than making a quick DIY solution